.github/workflows/fedora-interop.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM fedora:38
+RUN dnf -y update
+RUN dnf -y install perl
+RUN dnf -y install git tmt-all
+RUN dnf -y remove openssl
+RUN cd / && git clone https://gitlab.com/redhat-crypto/tests/interop
+RUN cd interop && git checkout 3def3aa0ef173b1479611909f13ac4abb604e62b
+VOLUME /openssl
```

Comment:
I guess this file is not needed anymore.

Reply:
yep, I'll remove it